ecommerce/courses/tests/test_publishers.py

```diff
@@ -79,25 +75,28 @@ def test_api_exception(self):
                 )
                 self.assertEqual(actual, self.error_message)
 
+    @responses.activate
     def test_api_error(self):
         """ If the Commerce API returns a non-successful status, an ERROR message should be logged. """
         status = 400
-        expected_msg = 'deadline issue'
-        api_body = {'non_field_errors': [expected_msg]}
 
-        self._mock_commerce_api(status, api_body)
+        self._mock_commerce_api(status)
         with LogCapture(LOGGER_NAME) as logger:
             actual = self.publisher.publish(self.course)
+            url = urljoin(self.site.siteconfiguration.commerce_api_url, f"courses/{self.course.id}/")
             logger.check(
                 (
                     LOGGER_NAME, 'ERROR',
-                    'Failed to publish commerce data for [{}] to LMS. Status was [{}]. Body was [{}].'.format(
-                        self.course.id, status, json.dumps(api_body))
+                    'Failed to publish commerce data for [{course}] to LMS. Error was {status} Client Error: ' \
```

Comment:
why do you need a backslash? BTW f-string could be used here as well, couldn't it?